src/modules/ie-objects/ObjectDetailPage.tsx

```diff
@@ -1031,7 +1040,16 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 	const renderExportDropdown = useCallback(
 		(isPrimary: boolean) => {
 			const icon = <Icon name={IconNamesLight.Export} aria-hidden />;
-			const label = tText('modules/ie-objects/object-detail-page___download-deze-krant');
+			const label = canDownloadNewspaper
+				? tText('modules/ie-objects/object-detail-page___download-deze-krant')
+				: tText('exporteer metadata');
+
+			const exportOptions =
+				canDownloadNewspaper && canDownloadMetadata
+					? [...NEWSPAPER_DOWNLOAD_OPTIONS(), ...METADATA_EXPORT_OPTIONS()]
+					: canDownloadNewspaper
+					? NEWSPAPER_DOWNLOAD_OPTIONS()
+					: METADATA_EXPORT_OPTIONS();
```

Comment:
Your code allows metadata export when the user does not have the canDownloadMetadata permission
I'm not a big fan of double tertiary operators

This code is a lot clearer and its less code
```suggestion
			const exportOptions = [];
			if (canDownloadNewspaper) {
				exportOptions.push(...NEWSPAPER_DOWNLOAD_OPTIONS());
			}
			if (canDownloadMetadata) {
				exportOptions.push(...METADATA_EXPORT_OPTIONS());
			}
```